src/metrics/dream_app.ml

```diff
@@ -0,0 +1,51 @@
+(* Most of this code is borrowed from https://github.com/mirage/prometheus/blob/master/app/prometheus_unix.ml which is under the Apache license *)
+
+open Prometheus
+open Prometheus_app
+
+module Unix_runtime = struct
```

Comment:
Could we move this module to `prometheus_app` and deduplicate some of the code? Unless there's some reason not to, in which case it would be good to explain with a comment.

Reply:
This code is moved to https://github.com/marigold-dev/prometheus-web